Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dapper;
+using Doppler.ContactPolicies.Data.Access.Core;
+using Doppler.ContactPolicies.Data.Access.Entities;
+
+namespace Doppler.ContactPolicies.Data.Access.Repositories.ContactPoliciesSettings
+{
+    public class ContactPoliciesSettingsRepository : IContactPoliciesSettingsRepository
+    {
+        private readonly IDatabaseConnectionFactory _databaseConnectionFactory;
+
+        public ContactPoliciesSettingsRepository(IDatabaseConnectionFactory databaseConnectionFactory)
+        {
+            _databaseConnectionFactory = databaseConnectionFactory;
+        }
+
+        #region public methods
+
+        public async Task<Entities.ContactPoliciesSettings> GetContactPoliciesSettingsAsync(string accountName)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+            var query =
```

Comment:
Can `query` variable be a constant?
```suggestion
            const string query =
```